crystal-map-processor/src/main/java/com/schwarz/crystalprocessor/model/entity/BaseEntityHolder.kt

```diff
@@ -56,9 +56,18 @@ abstract class BaseEntityHolder(val sourceElement: ISourceModel) : IClassModel b
     val interfaceTypeName: TypeName
         get() = ClassName(sourcePackage, interfaceSimpleName)
 
-    fun collectAllChildInterfaces(): List<TypeName> {
+    override fun hasFieldWithName(name: String): Boolean = fields.containsKey(name)
+    override fun hasFieldConstantWithName(name: String): Boolean = fieldConstants.containsKey(name)
+
+    fun collectAllSuperInterfaceNames(): List<TypeName> {
```

Comment:
I needed the new method for the fields, so I thought it would be helpful to change the name here to contain "Names". I hope that is ok...